installer/create-cluster-eks.sh

```diff
@@ -62,6 +62,40 @@ eksctl create nodegroup \
   --managed \
   --node-labels=dedicated=StrimziKafka
 
+# Attach EFS policy to Cloudflow roles
+cat <<EOF >describe-file-systems-policy.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": "elasticfilesystem:DescribeFileSystems",
+      "Resource": "*"
+    }
+  ]
+}
+EOF
+
+ROLE_NAME_KAFKA=$(aws iam list-roles | jq -r '.Roles[] | .RoleName' | grep "cloudflow" | grep "kafka")
+ROLE_NAME_STANDARD=$(aws iam list-roles | jq -r '.Roles[] | .RoleName' | grep "cloudflow" | grep "standa")
```

Comment:
`grep "cloudflow"` returns nothing on my side. The roles I see by running `aws iam list-roles | jq -r '.Roles[] | .RoleName'  | grep "standa"` have this format:
```eksctl-stavros31-nodegroup-standa-NodeInstanceRole-1JAE8PLMIXBHW```

Reply:
Also how about creating the security groups for the efs in order to fully automate this setup?
Btw I would also add a note about resources remaining around after deleting a cluster created with our scripts. 
This could be a follow-up PR ;)

Reply:
@skonto Let me take care of that filtering thingy so we can merge this bad boy

I agree with you on the automation part, I just feel bad every time we add lines of bash to the code base...

Reply:
@skonto I think I took care of your last comments, let's address the security groups thing on a follow-up PR. What do you think?

Reply:
>  I think I took care of your last comments, let's address the security groups thing on a follow-up PR. What do you think?
Sure thanks @eschizoid.

> I agree with you on the automation part, I just feel bad every time we add lines of bash to the codebase. If we continue towards that route I feel this is never going to stop...

I agree we need a better approach long term since for example no clean up is done.

Reply:
>Also it would be interesting to add support for Azure as well ;).
Happy to take care of that if we don't use bash :)